CHANGES.md

```diff
@@ -1,3 +1,6 @@
+# UNRELEASED
+- [NEW] Allow custom service name in CloudFoundry VCAP_SERVICES environment variable.
```

Comment:
This makes it sound like we're changing the content of the environment variable. Maybe:
>Added an extra `bluemix` method to the client builder allowing a custom service name to be used with the VCAP_SERVICES environment variable content.

Reply:
Will change.